remote-config.jsonc

```
@@ -123,6 +123,12 @@
         },
         {
           "message": "Did you know you can set environment variables with the `.ddev/.env` file?"
+        },
+        {
+          "message": "Use `ddev logs --follow --time` for insights on crashed containers or other misbehavior."
```

We try to be consistent about “running” commands:

```suggestion
          "message": "Run `ddev logs --follow --time` for insights on crashed containers or other misbehavior."
```